impl/integration/main_test.go

```diff
@@ -16,25 +16,37 @@ package integration
 
 import (
 	"context"
+	"flag"
 	"testing"
 
 	"github.com/google/keytransparency/core/integration"
 	"github.com/google/trillian/storage/testdb"
 )
 
+var (
+	generate = flag.Bool("generate", false, "Defines if test vectors should be generated")
+)
+
 // TestIntegration runs all KeyTransparency integration tests.
 func TestIntegration(t *testing.T) {
 	// We can only run the integration tests if there is a MySQL instance available.
 	testdb.SkipIfNoMySQL(t)
+	flag.Parse()
 
 	for _, test := range integration.AllTests {
 		t.Run(test.Name, func(t *testing.T) {
 			ctx, cancel := context.WithCancel(context.Background())
 			defer cancel()
-			env, err := NewEnv(ctx)
+			env, err := integration.NewEnv(ctx)
 			if err != nil {
 				t.Fatalf("Could not create Env: %v", err)
 			}
+			if *generate {
```

Comment:
In this configuration, `GenerateTestVectors` is it's own unit test and as such probably belongs outside the for loop, else we will generate the test vectors `len(AllTests)` times.

Reply:
Yeah, but anyway we run `NewEnv(ctx)` for each loop cycle, maybe we should move it outside too?